irods/data_object.py

```diff
@@ -90,6 +90,10 @@ def open(self, mode='r', finalize_on_close = True, **options):
         return self.manager.open(self.path, mode, finalize_on_close = finalize_on_close, **options)
 
     def chksum(self, **options):
+        """
+        See: https://docs.irods.org/4.2.10/doxygen/dataObjChksum_8h.html
```

Comment:
let's see if we can get a non-version-specific link... hmmm... just point to the source in 4-2-stable?  not sure...